### DISCUSSION TITLE
Welcome to OpenEHR-registry-template Discussions!

### DISCUSSION THREAD

**Discussion by ardihikaru** [bgsi-registry/OpenEHR-registry-template#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lamurian** [comment]
I checked in [Archetype designer](https://tools.openehr.org/), and some archetypes in our repo are outdated, so I've selectively updated archetypes in the `local` directory. On another note, have you been able to synchronize your workspace in Archetype designer with this repository? @mnardisasmita

**Reply by mnardisasmita** [reply]
noted @lamurian , last night i try synchronization it, but it give error message, but now is already done
